web_src/js/features/repo-issue-pr-status.js

```diff
@@ -0,0 +1,11 @@
+export function initRepoPullRequestCommitStatus() {
```

Comment:
This function doesn't look right. If you have many `repo/pulls/status.tmpl` on one page, they conflict with each other.

Reply:
This JS is for the PR box which is one per page, but I guess it wouldn't hurt making it compatible with multiples as well.